include/zephyr/drivers/adc.h

```diff
@@ -156,6 +156,18 @@ struct adc_channel_cfg {
 	 */
 	uint8_t current_source_pin[2];
 #endif /* CONFIG_ADC_CONFIGURABLE_EXCITATION_CURRENT_SOURCE_PIN */
+
+#ifdef CONFIG_ADC_CONFIGURABLE_VBIAS_PIN
+	/**
+	 * Output pins for the bias voltage.
+	 * This is only available if the driver enables this feature
+	 * via the hidden configuration option ADC_CONFIGURABLE_VBIAS_PIN.
+	 * The field is interpreted as bit mask, where each pin represents
+	 * on of the input pins. The actually mapping to the physical pins
+	 * depends on the driver itself.
```

Comment:
```suggestion
	 * The field is interpreted as a bitmask, where each bit represents
	 * one of the input pins. The actual mapping to the physical pins
	 * depends on the driver itself.
```

Reply:
I must have had a seizure while typing this. :worried:
Thanks for pointing it out.